tinyagentos/routes/agent_auth_requests.py

```
@@ -111,6 +111,18 @@ def _get_relationships(request: Request):
 
 
+async def _retire_request_notification(request: Request, request_id: str) -> None:
+    """Archive the bell notification for a now-decided auth request so it leaves
+    the active list. Best effort: never fails the decision."""
+    notifs = getattr(request.app.state, "notifications", None)
+    if notifs is None:
+        return
+    try:
+        await notifs.archive_by_source_ref("auth_requests", request_id)
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)